core/os/os.cpp

```diff
@@ -55,6 +55,10 @@ OS *OS::get_singleton() {
 	return singleton;
 }
 
+bool OS::prefer_meta_over_ctrl() {
+	return singleton->has_feature("macos") || singleton->has_feature("web_macos") || singleton->has_feature("web_ios");
```

Comment:
```suggestion
#if defined(MACOS_ENABLED) || defined(APPLE_EMBEDDED_ENABLED)
	return true;
#elif defined(WEB_ENABLED)
	return singleton->has_feature("web_macos") || singleton->has_feature("web_ios");
#else
	return false;
#endif
```

This should be enabled for iOS as well (when it was originally added, shortcuts were used only for editor, but this is not the case anymore). Also, there is no need to do this check s on non Apple/Web platforms, so it can be ifdef.